### DISCUSSION TITLE
OpenCV

### DISCUSSION THREAD

**Discussion by zombozo12** [mobile-dev-inc/Maestro#371] [Q&A] [answered]
Is it possible to integrate with OpenCV?

**Comment by axelniklasson** [comment, marked as answer]
Hi there @zombozo12! There are no plans to integrate OpenCV at this moment. Thanks for the suggestion though!

**Reply by zombozo12** [reply]
Sorry, I wasn't clear to say my thought. What I mean to integrate is by using current state of the Maestro project. Either using scripts with js (current availability), or create a workflows that take a screenshots then send it to 3rd party software to be processed.

**Reply by axelniklasson** [reply]
Ah I see. You could definitely use either JS and integrate via HTTP requests to OpenCV if that is possible or output the screenshots somewhere and have that piped into OpenCV via some post-test script. Nothing in the plans for first-class support in maestro though 🙂

**Reply by zombozo12** [reply]
But can Maestro receive what OpenCV send back and give some action based what OpenCV gave? Perhaps OpenCV services gave XY coordinates of the screen.
